### ISSUE TITLE
Referencing used competency models and other value spaces

### ISSUE THREAD

**Issue by stuartasutton** [jobdataexchange/Data-Modeling#15]
On today's deep dive webinar, the session closed with a question by Matt that we did not answer fully due to time. Since it is such a key question, I've opened this issue to capture the response:

>"Let’s also make sure to address how in the Schema+ we reference any used competency models. You may have covered this, so sorry if you did."

Competency models, as well as ontologies, taxonomies and flat enumerations (all controlled vocabularies or value spaces) are formally addressed in the JDX model by building off the [`schema.org/DefinedTerm`](https://pending.schema.org/DefinedTerm) which we call the `jdx:AnnotatedDefinedTerm`. So, whenever we refer to a term in a controlled value space, we are able to not only identify the term but also the term source. Here is an example from the schema.org documentation:

```
{
 "@context": "http://schema.org/",
 "@type": "DefinedTerm",
 "name": "Shoe Machine Operators and Tenders",
 "termCode": "51-6042.00",
 "description": "Operate or tend a variety of machines to join, decorate, reinforce, or finish shoes and shoe parts.",
 "inDefinedTermSet": "http://onetonline.org"
}
```
Here's the part of the model where this is handled:

<img width="1315" alt="addressing_valur_spaces" src="https://user-images.githubusercontent.com/2939046/53112943-46256b00-34f5-11e9-9c75-7445cff898a5.png">

Note also that should a term need to be scaled in some way, its `annotation` property can reference a `jdx:ScaleAnnotation` entity where such scaling on the term can be provided. 

Hope this addresses the question...always possible that I missed the point.

**Comment by philbarker** [comment]
@stuartasutton and I have discussed this over the last couple of days. After exploring an alternative, I agree with this approach. One point to note is that the `@id` of the AnnotatedDefinedTerm should not be the URI identifier for the term in vocabulary to which it points. (I think Stuart and I agreed on this, I'm putting it here partly to check on that!)

**Comment by stuartasutton** [comment]
Agreed. I have been working on this this morning and I am not certain that AnnotatedDefinedTerm itself should not actually be a blank node.

**Comment by DMSaunders** [comment]
Should this issue be closed?

**Comment by stuartasutton** [comment]
That's not a question I can answer, Danielle, since I don't know whether BH understands [Phil's point](https://github.com/jobdataexchange/Data-Modeling/issues/15#issuecomment-472100814) or [my point](https://github.com/jobdataexchange/Data-Modeling/issues/15#issuecomment-472113914). Since no one from the BH team doing the implementing responded here to the issue, it should probably not be closed and the issue added to the JDX team running agenda.
